samples/client/petstore/python-experimental/tests/test_pet_api.py

```diff
@@ -175,6 +175,36 @@ def test_separate_default_config_instances(self):
         pet_api2.api_client.configuration.host = 'someotherhost'
         self.assertNotEqual(pet_api.api_client.configuration.host, pet_api2.api_client.configuration.host)
 
+    def test_http_signature(self):
+        config = Configuration(
```

Comment:
Ah, this is a v2 test only.
Yes please fee free to add the test in either of the below folders
- samples/openapi3/client/petstore/python-experimental/tests (manual tests I think)
- samples/openapi3/client/petstore/python-experimental/test (autogen tests I think)
Either location should be fine because those files are not overwritten once they exist

Reply:
One thing I just discovered is that the directories don't seem to be merged by default.
I created a file samples/openapi3/client/petstore/python-experimental/tests/test_http_signature.py, and I thought by default it would be merged with the files under samples/client/petstore/python-experimental/tests.

Reply:
So the specs could be different. Those directories are generated by separate bash files. For now I prefer to keep them separate. Also because one is for v2 spec testing and one is for v3 spec testing. So they have different features.

Reply:
ok, sure. I have created a separate unit test file.